src/main/java/com/google/devtools/build/lib/analysis/starlark/StarlarkCustomCommandLine.java

```diff
@@ -655,12 +690,12 @@ public boolean equals(Object o) {
         return false;
       }
       VectorArg vectorArg = (VectorArg) o;
-      return features == vectorArg.features;
+      return features == vectorArg.features && Objects.equal(elementType, vectorArg.elementType);
     }
 
     @Override
     public int hashCode() {
-      return Integer.hashCode(features);
+      return Objects.hashCode(features, elementType);
```

Comment:
This is going to create a garbage array for the varargs and box the int. Instead:

```
return 31 * Integer.hashCode(features) + HashCodes.hashObject(elementType);
```

See `com.google.devtools.build.lib.util.HashCodes`.